backend/btrixcloud/operator.py

```diff
@@ -232,9 +231,15 @@ class CrawlStatus(BaseModel):
     restartTime: Optional[str]
     canceled: bool = False
 
-    # Execution Time -- updated on pod exits and at regular interval
+    # updated on pod exits and at regular interval
+    # Crawl Execution Time -- time crawl all pods have been running
+    # used to track resource usage
     crawlExecTime: int = 0
 
+    # Elapsed Exec Time -- time crawl has been running in at least one pod
+    # used for crawl timeouts
+    elapsedExecTime: int = 0
```

Comment:
I might suggest renaming this to something like `elapsedActiveCrawlTime` or something, to avoid further confusion about execution time since it's already not self-evident to everyone?